qcschema/dev/wavefunction/result_wavefunction.py

```diff
@@ -0,0 +1,78 @@
+"""
+The primary specified return  wavefunction quantities.
+"""
+
+result_wavefunction = {}
+
+# Orbitals
+result_wavefunction["return_result_orbitals_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_orbitals_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin orbitals in the AO basis",
+    "items": {"type": "number"}
+}
+
+# Density
+result_wavefunction["return_result_density_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin density in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_density_b"] = {
+    "type": "array",
+    "description": "The primary specified return beta-spin density in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+# Fock matrix
+result_wavefunction["return_result_fock_a"] = {
+    "type": "array",
+    "description": "The primary specified return alpha-spin Fock matrix in the AO basis",
+    "items": {"type": "number"}
+}
+
+
+result_wavefunction["return_result_fock_b"] = {
```

Comment:
For this what if we imply that any `fock_a` is the `return_result` request? Would cut down on the typing considerably.